### ISSUE TITLE
Error: Anomaly "Uncaught exception Failure("action op")."

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#18436]
#### Description of the problem

```bash
$ printf 'Set Ltac Debug. Ltac foo := idtac; idtac "bar"; idtac "baz". Goal True. foo.\nr "bar"\n' | coqtop
Welcome to Coq 8.18.0

Coq <
foo is defined

1 goal

  ============================
  True

Goal:

  ============================
   True


Going to execute:
foo

TcDebug (0) > Executed expressions: 0

Executed expressions: 1

Executed expressions: 2

Executed expressions: 3

bar
Toplevel input, characters 72-76:
> Set Ltac Debug. Ltac foo := idtac; idtac "bar"; idtac "baz". Goal True. foo.
>                                                                         ^^^^
Error: Anomaly "Uncaught exception Failure("action op")."
Please report at http://coq.inria.fr/bugs/.

Unnamed_thm <
```

cc @jfehrle 

#### Coq Version

8.18.0

**Comment by ejgallego** [comment]
cc @jfehrle

**Comment by jfehrle** [comment]
Thanks for the report.  This is a duplicate of #18067, which is fixed by #18068 in 8.19.

**Comment by JasonGross** [comment]
Neat, thanks!  Sorry I missed that
